openshift/e2e-tests-openshift.sh

```diff
@@ -241,6 +241,16 @@ function teardown() {
   delete_knative_openshift
 }
 
+function dump_openshift_ingress_state(){
+  echo ">>> routes.route.openshift.io:"
+  oc get routes.route.openshift.io -o yaml --all-namespaces
```

Comment:
Although this line would be dup of https://github.com/openshift/knative-serving/blob/4fc2deb829c15ba9fd2350f1726fa623a30eb874/test/e2e-common.sh#L332-L333, but collects both `routes.route.openshift.io` and `routes.serving.knative.dev` for sure.